src/undo_log/undo_log.c

```diff
@@ -246,10 +248,9 @@ static void _UndoLog_Rollback_Delete_Node
 	int seq_start,
 	int seq_end
 ) {
-	UndoOp *undo_list = ctx->undo_log;
 	for(int i = seq_start; i > seq_end; --i) {
```

Comment:
in this file some functions use the following loop construction:
`for(int i = seq_start; i > seq_end; --i) {`
others use:
`for(int i = 0; i < _count; i++) {`

Can we please consolidate to one pattern ?